_posts/en/newsletters/2018-10-16-newsletter.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Bitcoin Optech Newsletter #17"
+permalink: /en/newsletters/2018/10/16/
+name: 2018-10-16-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter briefly describes a proposal for splicing
+Lightning Network payment channels, links to videos and transcripts of
+talks from the Edge Dev++ training sessions, and summarizes some
+transcripts made during last week's CoreDev.tech conference.
```

Comment:
I prefer the word 'event' to 'conference' here. Conference implies organized schedules and presentations.

Reply:
I think "conference" implies people *conferring* much more than it implies organized schedules and presentations.  :-)  But with that grouse out of the way, I'll change it throughout the document, as arguing over terms is not productive.